reference/regexp.go

```diff
@@ -134,10 +134,10 @@ func optional(res ...string) string {
 	return `(?:` + strings.Join(res, "") + `)?`
 }
 
-// repeated wraps the regexp in a non-capturing group to get one or more
-// matches.
-func repeated(res ...string) string {
-	return `(?:` + strings.Join(res, "") + `)+`
+// zeroOrMore wraps the expression in a non-capturing group that can occur
+// zero or more times.
+func zeroOrMore(res ...string) string {
```

Comment:
🚲 🏠  🎨 `func anyTimes`? `func star`?

Reply:
Naming is hard! I'm fine either way, so changed it to `anyTimes`